iiop-openjdk/src/main/resources/schema/jboss-as-iiop-openjdk_1_0.xsd

```diff
@@ -46,15 +46,16 @@
             </xs:documentation>
         </xs:annotation>
         <xs:sequence>
+            <xs:element name="properties" minOccurs="0" maxOccurs="1" type="genericPropertiesType"/>
```

Comment:
Order is not controlled there.  at least not in this case.
When it comes to persisting attributes in context of a resource, order is as follows:
1) persist all attributes that don't belong to any attribute group and persist to xml attributes using the order attributes ware added to parser definition (not resource definition)
2) persist all attributes that don't belong to any attribute group and don't persist to xml attributes but to elements in order that ware added to parser definition
3) persist attribute groups (each group = xml element) and persist based on 1) & 2) 

so in short for this case, properties attribute doesn't belong to any attribute group so it is persisted directly after all other non grouped attributes on a resource.
In this case this is first child of the resource element.